AppCenterReactNativeShared/android/build.gradle

```diff
@@ -34,8 +34,8 @@ allprojects {
         defaultConfig {
             minSdkVersion 16
             targetSdkVersion 26
-            versionCode 44
-            versionName '2.3.0'
+            versionCode 45
+            versionName '2.5.0'
```

Comment:
Why do we skip 2.4.0 here?

Reply:
In last release, the android sdk was not updated, so I reverted all android build.gradle updates. But it seems they should not be skipped. They was updated by running wrapper script, and the wrapper script is for the npm package version, JavaScript layer.